### ISSUE TITLE
Node daemon multiple queries

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#90]
Originally reported of Google Code with ID 34.

```
Opening bug for tracking multiple queries at the same time functionality in
the node daemon.
```
Originally added on 2008-07-21 13:26:17 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
This was implemented and is working in 2.0b1.
```
Originally added on 2009-01-28 15:41:03 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:07 +0000 UTC.

Changed State: Released
